examples/project/dialog_box/test_2_input_field.py

```diff
@@ -34,3 +34,32 @@ def test_num_input():
     response = int(run_dialog_box(dbx))
     set_message(f"Entered number {response}")
     assert response == TEST_NUM, "The entered number is not correct"
+
+
+@pytest.mark.case_name("Radiobutton")
```

Comment:
Need to move the radiobutton and checkbox examples to `test_3_choice_control.py`

Reply:
Ready